cdap-client/src/main/java/co/cask/cdap/client/AuthorizationClient.java

```diff
@@ -203,8 +207,12 @@ private HttpResponse executeRequest(HttpRequest request, int... allowedErrorCode
     HttpResponse response = restClient.execute(request, config.getAccessToken(), allowedErrorCodes);
     if (HttpURLConnection.HTTP_FORBIDDEN == response.getResponseCode()) {
       // TODO:(CDAP-5302) Include the logged in username here
-      throw new UnauthorizedException(String.format("Unauthorized to perform %s %s with body %s",
-                                                    request.getMethod(), request.getURL(), request.getBody()));
+      InputSupplier<? extends InputStream> requestBody = request.getBody();
+      String msg = requestBody == null ?
+        String.format("Unauthorized to perform %s %s", request.getMethod(), request.getURL()) :
+        String.format("Unauthorized to perform %s %s with body %s",
```

Comment:
Since body can be huge, it is better not to log it here. Also, is there a reason for an operation to be unauthorized due to body only?